codecov.yml

```diff
@@ -0,0 +1,4 @@
+coverage:
+  range: 50..90
+ignore:
+  - "ebmlite/tools"
```

Comment:
just curious, why exclude the tools files? they do have (basic integration) tests

Reply:
They're not currently tested at all, and as far as I know there's no plan to test them.